static/src/stylesheets/module/commercial/glabs/_hosted.scss

```diff
@@ -188,8 +184,8 @@ $leffeColor: #ffffff;
         }
 
         &:after {
-            top: 16px;
-            left: 16px;
+            top: 50%;
+            left: 25px;
```

Comment:
might be helpful to add a comment as to why this is `25px` as it looks a bit magic in the diff